lib/matplotlib/backends/backend_wx.py

```diff
@@ -1236,6 +1236,7 @@ def new_figure_manager_given_figure(num, figure):
     figmgr = frame.get_figure_manager()
     if matplotlib.is_interactive():
         figmgr.frame.Show()
+        figure.canvas.draw_idle()
```

Comment:
Sorry, now re-reading this and understanding it correctly.  I think you are reffering to the issue fixed by https://github.com/matplotlib/matplotlib/pull/3905 so I am inclined to leave this as-is unless a Wx users says otherwise.